### PR TITLE
Action endpoint integration

### PR DESCRIPTION
**Pull request by agalperina**
Action endpoint integration

### CODE REVIEW

File: frontend/src/pages/simulationWindow.js [hunk 1]
```diff
@@ -23,22 +23,26 @@ function SimulationWindow() {
...
-    initialReflection: { visited: false, completed: true, pageNumber: 2, html: (<Reflection
+    initialReflection: { visited: true, completed: true, pageNumber: 2, html: (<Reflection
       content_url="/scenarios/initialReflection" res_url="/scenarios/initialReflection/response" nextPageID="initialAction" prevPageID="projectAssignment" title="Reflect on Initial Information"/>) },
-    initialAction: { visited: false, completed: false, pageNumber: 3, html: (<InitialAction />) },
+    initialAction: { visited: true, completed: true, pageNumber: 3, html: (<Action
+      content_url="/scenarios/initialAction" nextPageID="gatheredInformation" prevPageID="initialReflection" title="Initial Action"/>) },
     gatheredInformation: { visited: false, completed: false, pageNumber: 4, html: (<GatheredInformation />) },
     stakeholders: { visited: false, completed: true, pageNumber: 5, html: (<Stakeholders />) },
-    middleReflection: { visited: false, completed: true, pageNumber: 6, html: (<Reflection
+    middleReflection: { visited: true, completed: true, pageNumber: 6, html: (<Reflection
```
Comment (rasdy) [inline frontend/src/pages/simulationWindow.js]: The reflection and action pages should have `visited: false`

File: frontend/src/pages/action.js [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+    await axios({
+      url: BASE_URL + content_url,
+      method: 'put',
+      data: {
+        scenarioID: scenarios.currentScenarioID,
+        studentID: STUDENT_ID,
+        data: request_data
+      }
+    });
+  }
```
Comment (rasdy) [inline frontend/src/pages/action.js:80]: When I select the "Approve" option for the final action, It sends `data: {3: "3"}` even though question 3 doesn't have an option with id: 3

**Review by rasdy — CHANGES_REQUESTED** [review]
Found two issues

File: frontend/src/pages/action.js (hunk 2 above)
Comment (rasdy) [reply frontend/src/pages/action.js:80]: My bad, this is incorrect. I was looking at the initial action GET.

File: frontend/src/pages/simulationWindow.js (hunk 1 above)
Comment (agalperina) [reply frontend/src/pages/simulationWindow.js]: Fixed thanks!

**Review by rasdy — APPROVED** [review]

**Pull request merged**